<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="openFileBtn">
      <property name="text">
       <string>Open file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="readFileBtn">
      <property name="text">
       <string>read file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="parseFileBtn">
      <property name="text">
       <string>parse file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chckbxSaveToFile">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>save out to file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>273</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
